### ISSUE TITLE
better image alt handling

### ISSUE THREAD

**Issue by pkra** [Speech-Rule-Engine/speech-rule-engine#635]
Follow up to #414.

It seems missing alt text for images leads to images being ignored. It seems better to follow how alt on images works in general --  if alt is empty, ignore it entirely (i.e., it's decorative) and, if alt is missing, use some fallback (e.g., dummy text).

**Comment by pkra** [comment]
There's a commit marking this as fixed. (FWIW I think that's still surprising behavior for users.)
